src/graphql/Workers.graphql

```diff
@@ -13,16 +13,40 @@ type WorkerCompact {
   workerType: String!
 }
 
+# Definition of a worker
 type Worker {
+  # Provisioner ID name.
   provisionerId: String!
+
+  # Worker type name.
   workerType: String!
+
+  # Identifier for group that worker who executes this run is a part of,
```

Comment:
Nit: end of sentence, end with period. Make next line a sentence with capitalized letter.